app/javascript/shared/utils/PerformanceUtils.re

```diff
@@ -3,7 +3,7 @@ open Webapi.Dom;
 [@bs.send] external performanceNow: Window.performance => float = "now";
 
 /* Logs the time (in ms) it takes to run a function, using the DOM Performance API. */
-let measure = f => {
+[@dead "measure"] let measure = f => {
```

Comment:
@harigopal yes I think there should be something for this kind of interactive use.
Something like `[@live]` or so to prevent a valur from being considered dead.

Reply:
Added `[@live]` support here: https://github.com/cristianoc/genType/pull/279.